### ISSUE TITLE
Overfitting on the training set of BurstSR

### ISSUE THREAD

**Issue by greatlog** [akshaydudhane16/BIPNet#1]
Hi, thanks for your wonderful work. I am also working on BurstSR and your codes have helped me a lot. 

While I have some problems when I try to fine-tune an SR model on the BurstSR dataset. I firstly train the SR model on the Synthetic dataset and everything is OK. Then I fine-tune this model on the BurstSR dataset. The training loss keeps decreasing while the validation PSNR only grows at the beginning and drops gradually after it achieves the best result. It seems that the training set is overfitted. 

Have you met the same problem? I would appreciate it a lot if you could help me figure this out.

Thanks.

**Comment by akshaydudhane16** [comment]
We did not encountered such problem. May be try reducing learning rate, batch size.

**Comment by RenLongWu** [comment]
> Hi, thanks for your wonderful work. I am also working on BurstSR and your codes have helped me a lot.
> 
> While I have some problems when I try to fine-tune an SR model on the BurstSR dataset. I firstly train the SR model on the Synthetic dataset and everything is OK. Then I fine-tune this model on the BurstSR dataset. The training loss keeps decreasing while the validation PSNR only grows at the beginning and drops gradually after it achieves the best result. It seems that the training set is overfitted.
> 
> Have you met the same problem? I would appreciate it a lot if you could help me figure this out.
> 
> Thanks.

Have you soveld this problem? Thanks!

**Comment by Egkang-Luis** [comment]
Have you solved this problem? Thanks
